Example/CollectionViewController.swift

```diff
@@ -38,23 +38,28 @@ final class CollectionViewController: UICollectionViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        self.collectionView!.register(
+        guard let collectionView = self.collectionView, let flowLayout = collectionView.collectionViewLayout as? UICollectionViewFlowLayout else { return }
+        collectionView.register(
             UINib(nibName: "CollectionToolCell", bundle: nil),
             forCellWithReuseIdentifier: "CollectionToolCell"
         )
 
+
+        collectionView.register(UINib(nibName: "CollectionViewHeaderView", bundle: nil), forCellWithReuseIdentifier: "CollectionViewHeaderView")
+        flowLayout.headerReferenceSize = CGSize(width: 375, height: 25)
```

Comment:
@jessesquires - Note it seems that in order to have the supplementary view delegate calls called I had to explicitly set the `flowLayout`'s `headerReferenceSize`.  We probably want to ensure that a user doesn't have to do this somehow.  This bug also could have been introduced maybe due to the refactoring happening in the Collection View types.

Reply:
This was always a problem I think. Related: #49 

You either set this property, or implement `FlowLayoutDelegate`

Reply:
opened #65